res/controllers/Numark-Mixtrack-Platinum-scripts.js

```diff
@@ -1454,3 +1713,10 @@ MixtrackPlatinum.shiftToggle = function (channel, control, value, status, group)
         MixtrackPlatinum.head_gain.unshift();
     }
 };
+
+// simple debugging function
+MixtrackPlatinum.dbg = function () {
+    if (Debug) {
+        console.log.apply(null, arguments);
```

Comment:
Any reason for not using the [spread operator](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Spread_syntax#syntax) in the function def and function call?